ws-nextjs-app/pages/[service]/av-embeds/README.md

```diff
@@ -0,0 +1,51 @@
+# AV
+
+## Description
+
+AV Embeds routes deliver BBC media to thrid party websites.
+
+## BFF Response
+
+BFF details for AV Embed pages can be found here:
+[BFF Send Response](https://github.com/bbc/fabl-modules/tree/main/modules/application/simorgh-bff#av-embed)
+
+## How routes are handled
+
+There are no direct app routes connected to this component, because av-embeds are inteded to be served as a raw piece of HTML outside any kind of framework. (In other words: There is no .page.tsx file exposed here, because this component is not intended to be run within our Next.js app, it's intended to be embedded into thid party sites as raw HTML.)
```

Comment:
```suggestion
There are no direct app routes connected to this component, because av-embeds are intended to be served as a raw piece of HTML outside any kind of framework. (In other words: There is no .page.tsx file exposed here, because this component is not intended to be run within our Next.js app, it's intended to be embedded into third party sites as raw HTML / within an iFrame)
```